blog/html_files/regressions-in-dbt.html

```diff
@@ -0,0 +1,306 @@
+
+<pre><code>
+{# Transform feature list into a wide table with one row, and each coefficient in its own column #}
+WITH pivoted_model_coefficients as (
+  SELECT
+    {%- for feature in feature_list %}
+    any_value(
+    case when feature_name = '{{feature}}' then coefficient end
+    ) as {{feature}}{%- if not loop.last %},{%- endif -%}
+    {% endfor %}
+  FROM {{ ref('saved_coefficients_seed') }}
+),
+</code></pre>
+
+<p></p>
+<p><strong>Apply Regression Formula</strong>
+<br>    
+In the next step, we’ll prepare our data for prediction by joining in the wide-format coefficient values, and finding the linear combination of features & coefficients. I won’t go super in-depth on the math here, but it’s helpful to have a basic understanding of how a logistic regression works under the hood in order to understand the code below. </p>
+<p>Similar to a linear regression, we’ll first find a linear combination of features and their coefficients with the basic formula of:</p>
+<p>&beta;<sub>0&nbsp;</sub> + &beta;<sub>1&nbsp;</sub>x<sub>1</sub> + ... + &beta;<sub>k&nbsp;</sub>x<sub>k</sub></p>
+<p>Where “x” represents a particular feature, “&beta;” represents the respective coefficient for the feature, and &beta;<sub>0&nbsp;</sub>​ is the intercept:</p>
+
+<pre><code>
+{# Create the linear combination of ((feature_value_1 * coefficient_value_1) + (feature_value_2 * coefficient_value_2) + ...etc.) for all saved features and their coefficients #}
+
+sigmoid_prep as (
+  SELECT 
+    p.*,
+    (
+    {% for feature in feature_list %}
+      {% if feature == 'intercept' -%}
+      c.{{feature}}
+      {%- else %}
+      (p.{{feature}} * c.{{feature}})
+      {%- endif %}{%- if not loop.last %} +{%- endif -%}
+    {% endfor %}
+    ) as linear_combination
+  FROM {{ ref('prep_new_data_for_prediction') }} as p
+    --Join in model coefficients
+    JOIN pivoted_model_coefficients as c
+      ON 1=1
+)    
+</code></pre>
+
+<p>From here, we’re ready to apply the sigmoid function. As I mentioned before, the code above would effectively work as-is if you were working on a linear regression. The main differentiation between the two models is that linear regression predicts a continuous outcome, while logistic regression predicts a probability of the outcome belonging to a particular class, and outputs probability scores that are bound between 0 and 1.</p>
+
+<p>In order to transform our linear combination above, we can use the sigmoid function to transform the output variable into something that represents a probability. The sigmoid function is defined as:</p>
+<p>\( \sigma(x) = \frac{1}{1 + e^{-x}} \)</p>
+
+<p>Where “x” represents the linear combination of features and coefficients outlined above.</p>
+<p>At this point, all we need to do is pass through the value we calculated in the previous step to the sigmoid function and we’ll arrive at our final probability outputs between 0 and 1:</p>
+<pre><code>
```

Comment:
```suggestion
<pre><code class="language-python">
```

This might work but I'm not sure